scripts/o3de/tests/test_utils.py

```diff
@@ -7,9 +7,45 @@
 #
 
 import pytest
+import pathlib
+import psutil
+import subprocess
```

Comment:
I could be wrong, but I don't think you actually need to import `subprocess` to mock/patch it

Reply:
nope, turns out it's needed to fetch definitions of `subprocess.Popen`. See here:

![image](https://user-images.githubusercontent.com/112996779/228668621-d716cf66-8408-4e9d-8033-13bb92001cb7.png)

Reply:
if I replace it to use `subproc_popen_patch` instead, the test works. But for unit tests with mocking, I'm not sure that's what we actually want.